src/app/story/storyrow/storyrow.component.ts

```diff
@@ -87,6 +87,7 @@ export class StoryRowComponent implements OnInit, AfterViewInit {
       case('KojiBuild'):
         return 'pficon-build';
       case('Advisory'):
+      case('ContainerAdvisory'):
         return 'pficon-security';
```

Comment:
I think a return statement for Advisory is missing.

Reply:
This is correct. This is basically saying, if the value is `Advisory` or `ContainerAdvisory`, return `'pficon-security'`.

Reply:
cool